### PR TITLE
[docs][l] Automation guide. Epic: https://github.com/datahq/pm/issues/78

### PR DESCRIPTION
**Pull request by AcckiyGerman**
Related to https://github.com/datahq/pm/issues/78

### CODE REVIEW

File: docs/automation.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+```json
+{
+  "name": "package_name",
+  "title": "package_title",
+  "description": "description",
+  "license": "ODC-PDDL-1.0",
+  // hope soon the readme will be grabbed automatically
+  "readme": "the very long string of readme, copy of readme.md",
+
+  // other non-necessary fields like:
```
Comment (zelima) [inline docs/automation.md]: maybe "non-required" is a better word here

File: docs/automation.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+`.datahub/datapackage.json` file is not required, though it is used to store the dataset metadata (and to describe intermediate resources if you need them) in the package.
+So create `.datahub/datapackage.json` file. Here is example for you:
+
+```json
+{
+  "name": "package_name",
+  "title": "package_title",
+  "description": "description",
+  "license": "ODC-PDDL-1.0",
+  // hope soon the readme will be grabbed automatically
```
Comment (zelima) [inline docs/automation.md]: Can we remove this?

Comment (AcckiyGerman) [reply docs/automation.md]: removed

File: docs/automation.md (hunk 1 above)
Comment (AcckiyGerman) [reply docs/automation.md]: fixed

**Pull request merged**